src/pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -277,13 +275,18 @@ def _adjust_batch_size(
         rank_zero_info(f"Batch size {batch_size} {desc}, trying batch size {new_size}")
 
     if trainer.state.fn == "fit":
+        from pytorch_lightning.trainer.supporters import CombinedLoader
+
         if trainer.train_dataloader is None:
             trainer.reset_train_dataloader()
 
         assert trainer.train_dataloader is not None
         # TODO: should we check val_dataloaders here too?
+
+        assert isinstance(trainer.train_dataloader, CombinedLoader)
```

Comment:
maybe the assert above can be deleted then?

Reply:
you're right.